### PR TITLE
Create one branch pipeline

### PR DESCRIPTION
**Pull request by alerickson**
# PR Summary
Move build and release pipelines to OneBranch in Azure DevOps.

### CODE REVIEW

File: .config/CredScanSuppressions.csk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+      {
+        "file": "Test/PublishPSResource.Tests.ps1",
+        "_justification": "Auto generated fake key"
+      }
+    ]
+  }
```
Comment (anamnavi) [inline .config/CredScanSuppressions.csk]:
```suggestion
  }
  
```

File: .config/CredScanSuppressions.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+      {
+        "file": "Test/PublishPSResource.Tests.ps1",
+        "_justification": "Auto generated fake key"
+      }
+    ]
+  }
```
Comment (anamnavi) [inline .config/CredScanSuppressions.json]:
```suggestion
  }
  
```

File: .config/tsaoptions.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://msazure.visualstudio.com",
+    "projectName": "One",
+    "areaPath": "One\\MGMT\\Compute\\Powershell\\Powershell\\PowerShell Core",
+    "notificationAliases": [
+        "adityap@microsoft.com",
+        "americks@microsoft.com",
+        "annavied@microsoft.com"
+    ]
+}
```
Comment (anamnavi) [inline .config/tsaoptions.json]:
```suggestion
}

```

File: global.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "6.0.418"
+    }
+}
```
Comment (anamnavi) [inline global.json]:
```suggestion
}

```

File: src/PowerShellGet.psd1 [hunk 5]
```diff
@@ -8,7 +8,7 @@
 
 @{
     RootModule = 'PowerShellGet.psm1'
-    ModuleVersion = '3.0.23'
+    ModuleVersion = '3.0.222'
```
Comment (anamnavi) [inline src/PowerShellGet.psd1]: why do we update this just for pipeline migration?

**Review by anamnavi — CHANGES_REQUESTED** [review]

File: .config/CredScanSuppressions.csk (hunk 1 above)
Comment (adityapatwardhan) [reply .config/CredScanSuppressions.csk]: Dont think you need the .csk file

File: .pipelines/PowerShellGet-Official.yml [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+      disableLegacyManifest: true
+      sbom:
+        enabled: true
+        packageName: Microsoft.PowerShell.PowerShellGet
+      codeql:
+        compiled:
+          enabled: true
+      asyncSdl: # https://aka.ms/obpipelines/asyncsdl
+        enabled: true
+        forStages: [Build]
```
Comment (adityapatwardhan) [inline .pipelines/PowerShellGet-Official.yml]:
```suggestion
        forStages: [stagebuild]
```

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]
Did you verify the built module actually works? The files are signed etc?

**Comment by alerickson** [comment]
@adityapatwardhan files are signed and pkg is working

File: global.json [hunk 7]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (adityapatwardhan) [inline global.json]: We dont need this file

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Review by anamnavi — APPROVED** [review]

**Pull request merged**